File: backend/src/main/java/ipeps/pwd/wallet/payload/updatePayload/ContactUpdatePayload.java
package ipeps.pwd.wallet.payload.updatePayload;

import ipeps.pwd.wallet.entity.Address;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;
import java.util.UUID;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class ContactUpdatePayload {
    private UUID contact_id;
    private String firstname;
    private String lastname;
    private String email;
    private String phone;
    private List<Address> addresses;
}

File: frontend/src/app/modules/contact/model/payload/contact.update.payload.ts
import {PayloadInterface} from "@shared/model";

//payload = donnée transmise pour créer contact
export interface ContactUpdatePayload extends PayloadInterface{
  contact_id: string;
  lastname: string;
  firstname: string;
  email: string;
  phone: string;
  address: string;

  deleted:boolean;
  deletedAt:Date;
  deletedBy:string;
}
